### DISCUSSION TITLE
want use macro to have a shortcut for most used strings like <br />

### DISCUSSION THREAD

**Discussion by Imperius83** [microsoft/vscode-discussions#1007] [Extension Development QnA]
Hi everybody,
i'am new in lerning programming. Aktually i lern PHP an there are some strings, i realy often use like " < b r  
 / > ". VSC did not give me a offer while iam writing this. I think i need to use macro. Is there a way to use macro in VSC? Is there a other way to have a quick way to write so unconftable strings like "< b r   / > "   ?

Thank you.

**Comment by gjsjohnmurray** [comment]
Please review the pinned thread at https://github.com/microsoft/vscode-discussions/discussions/5 for advice about the current purpose of this forum, and for suggestions of other resources.
